packages/mobile/src/components/FeeDrawer.tsx

```diff
@@ -50,9 +50,12 @@ export default function FeeDrawer({
   const { t } = useTranslation(Namespaces.sendFlow7)
   const [expanded, setExpanded] = useState(false)
 
+  const allowExpanded = !!totalFee && !totalFee.isZero()
```

Comment:
I used this to prevent expanding the fees dropdown drawer when the total fee is 0 or undefined